Content.Client/PowerCell/PowerChargerVisualizerSystem.cs

```diff
@@ -29,7 +29,7 @@ protected override void OnAppearanceChange(EntityUid uid, PowerChargerVisualsCom
             args.Sprite.LayerSetVisible(PowerChargerVisualLayers.Light, true);
         }
         else
-            // 
+            //
```

Comment:
```suggestion
```
Probably either remove this or add a comment if one was intended?

Reply:
This one was from a failed patch application. Goodbye!